scripts/error_codes.py

```diff
@@ -207,6 +207,7 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
         "2961", # SMTChecker, covered by CL tests
         "6240", # SMTChecker, covered by CL tests
         "9576", # SMTChecker, covered by CL tests
+        "7126", # SMTChecker, analysis without callback (not sure how to cover this)
```

Comment:
I might have missed something, but I've seen `m_readFile` field being null for `CompilerStack` only in tests. 

In production code it's declared once, in `StandardCompiler::compileSolidity` function.

I am not sure what to do with that.